### DISCUSSION TITLE
1.5.1-beta4

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#518] [Обновления]
## Список изменений
- Добавлена поддержка Google Drive (только публичные ссылки, например: `https://drive.google.com/file/d/FILE_ID`)
- Добавлена поддержка Banned Video (banned.video)
- Добавлена поддержка корп. видео для `player.vimeo.com` (поддержка параметра app_id). Для использования перевода на `player.vimeo.com` нужно полностью отключение CSP.
- Добавлена поддержка Facebook (работает только с ссылками, которые содержат `/video/` или `/reel/`). Для использования перевода нужно полностью отключение CSP.
- Немного почищен код от @SashaXser в #508
- Улучшение очистки текста для YouTube от @SashaXser в #508

## What's Changed
* Удаление/комментирование неиспользуемого кода by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/508
* Более продуманная очистка by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/512

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.1-beta3a...1.5.1-beta4

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.1-beta4'>1.5.1-beta4</a>.</em>

**Comment by ilya-corneli** [comment]
Спасибо огромное!

**Comment by NullVerdict** [comment]
Можно было бы и dev.epicgames.com добавить, но там приходится декодировать base64 чтобы получить плейлист DASH-MPD, но ведь его нужно куда-то заливать.

**Reply by ilyhalight** [reply]
Надо будет посмотреть

**Comment by selincodes** [comment]
Приветствую. По поводу Google Drive. Показывает: "Возникла ошибка при переводе, попробуйте позже" Видео публичное по ссылке `https://drive.google.com/file/d/FILE_ID` И всеравно такая ошибка. 
Вот видео: `https://drive.google.com/file/d/1N_7tcGfYDM4hqEOqVoKSluksjZy7c1Cs`

**Reply by ilyhalight** [reply]
у меня работает
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/62353659/2cb24f98-83bf-4000-9c76-62c2dce253f0)

**Comment by Ruhas80** [comment]
Всем привет! В первую очередь хочу сказать спасибо за такой инструмент! Меня интересует такой вопрос, где в коде можно делать регулировки громкости оригинального звука видео? Спасибо!

**Reply by ilyhalight** [reply]
> Всем привет! В первую очередь хочу сказать спасибо за такой инструмент! Меня интересует такой вопрос, где в коде можно делать регулировки громкости оригинального звука видео? Спасибо!

Вы можете регулировать громкость прямиком из меню с помощью слайдеров (появляются после активации перевода) и авто громкость в настройках расширения.
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/62353659/1696bcc6-abda-400d-b786-7d3c3c4b8e0b)

**Reply by Ruhas80** [reply]
Я делаю шаблон на Зеннопостере для автоматизации, и хотелось бы это все найти в самом коде.
